roxctl/netpol/generate/generate.go

```diff
@@ -81,8 +61,8 @@ func (cmd *NetpolGenerateCmd) ShortText() string {
 	return "(Technology Preview) Recommend Network Policies based on deployment information."
 }
 
-// LongText provides long command description
-func (cmd *NetpolGenerateCmd) LongText() string {
+// longText provides long command description
+func (cmd *NetpolGenerateCmd) longText() string {
```

Comment:
Out of curiosity: Is there a reason this was made private?

Reply:
This function was used by two command clones - the current one `netpol generate` and the deprecated one `generate netpol`. Now it is removed, but I made that private before following the IDE's suggestion.